lib/src/collect.dart

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2014, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library coverage.collect;
+
+import 'dart:async';
+
+import 'devtools.dart';
+import 'util.dart';
+
+const _retryInterval = const Duration(milliseconds: 200);
+
+Future<Map> collect(String host, int port, bool resume, bool waitPaused,
```

Comment:
Ooo! Yes.

Create a pull request which just exports this guy from lib/coverage.dart – is easy